projects/packages/sync/src/class-functions.php

```diff
@@ -721,4 +721,20 @@ public static function get_loaded_extensions() {
 	public static function get_jetpack_connection_active_plugins() {
 		return ( new Manager() )->get_connected_plugins();
 	}
+
+	/**
+	 * Return the list of active sync modules.
+	 *
+	 * @since $$next_version$$
```

Comment:
@zinigor Is this good enough to get a version in the next release? I checked similar functions like `get_jetpack_connection_active_plugins` above and they have not been replaced :thinking:

Reply:
It would need to be `@since $$next-version$$` : https://github.com/Automattic/jetpack/blob/trunk/docs/coding-guidelines.md#deprecating-code
(I made a similar mistake when answering that question recently - p1723032434387099-slack-CDLH4C1UZ)

Reply:
Ah, probably too late now (although it doesn't matter either way) - a PR to support the underscore has just been merged: https://github.com/Automattic/jetpack/issues/38760